persistence/persistence-mybatis/src/main/java/org/cbioportal/persistence/mybatis/StructuralVariantMyBatisRepository.java

```diff
@@ -46,14 +50,24 @@ public class StructuralVariantMyBatisRepository implements StructuralVariantRepo
     @Override
     public List<StructuralVariant> fetchStructuralVariants(List<String> molecularProfileIds,
                                                            List<String> sampleIds,
-                                                           List<Integer> entrezGeneIds) {
+                                                           List<Integer> entrezGeneIds, 
+                                                           List<StructuralVariantIdentifier> structuralVariants) {
```

Comment:
```suggestion
                                                           List<StructuralVariantIdentifier> structuralVariantIds) {
```

Reply:
Renamed in [4c93d75](https://github.com/cBioPortal/cbioportal/commit/4c93d75879595db2e4add0ff8452a5647b540756)